pycbc/filter/qtransform.py

```diff
@@ -337,20 +427,29 @@ def qtransform(fseries, Q, f0):
     # apply window to fft
     # normalize and generate bi-square window
     norm = np.sqrt(315. * qprime / (128. * f0))
-    windowed = fseries[start:end].numpy() * (bisquare(window_size) * norm)
+    windowed = fseries[start:end] * (bisquare(window_size) * norm)
+
+    # choice of output sampling rate
+    output_sampling = sampling # Can lower this to highest bandwidth
+    output_samples = int(dur * output_sampling)
 
     # pad data, move negative frequencies to the end, and IFFT
     padded = np.pad(windowed, padding(window_size, output_samples), mode='constant')
     wenergy = npfft.ifftshift(padded)
 
     # return a 'TimeSeries'
     wenergy = FrequencySeries(wenergy, delta_f=1./dur)
-    cenergy = TimeSeries(zeros(output_samples, dtype=np.complex128),
+    tdenergy = TimeSeries(zeros(output_samples, dtype=np.complex128),
                             delta_t=1./sampling)
-    ifft(wenergy, cenergy)
-    energy = cenergy.squared_norm()
-    medianenergy = np.median(energy.numpy())
-    norm_energy = energy / float(medianenergy)
+    ifft(wenergy, tdenergy)
```

Comment:
You also added in an extra line, and removed the usage of a method for a slower and more complicated function.